### PR TITLE
Ot124 33

### PR DESCRIPTION
**Pull request by juanpa0011**
- Add Auth "Login" and "ValidateToken"
- User does not exist "Yet" - MockUp consider to have "email" and "role"

### CODE REVIEW

**Comment by dhsosa98** [comment]
Perfect, but you need also the role in the ValidateToken middleware and set it in the req.role for to handle it in the autorization middleware in [Authorize admin middleware - OT124 52](https://github.com/alkemyTech/OT124-SERVER/pull/17/files)

File: src/validations/entriesSchema.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const yup = require("yup");
+
+const entriesCreatorSchema = yup.object().shape({
+  name: yup.string("Name must be a string").required("Name is required"),
+  content: yup.string("Content must be a string").required("Content is required"),
+  image: yup.string("Content must be a string").required("Content is required"),
```
Comment (rafalynch) [inline src/validations/entriesSchema.js:6]:
```suggestion
  image: yup.string("Image must be a string").required("Image is required"),
```

File: src/validations/entriesSchema.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  type: yup.string("Content must be a string").required("Content is required"),
```
Comment (rafalynch) [inline src/validations/entriesSchema.js:7]:
```suggestion
  type: yup.string("Type must be a string").required("Type is required"),
```

File: src/validations/entriesSchema.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  categoryId: yup.number("ContentId must be a number").required("ContentId is required").positive().integer()
```
Comment (rafalynch) [inline src/validations/entriesSchema.js:8]:
```suggestion
  categoryId: yup.number("CategoryId must be a number").required("CategoryId is required").positive().integer()
```

**Pull request closed without merging**